chronograf/server/influx_test.go

```diff
@@ -10,7 +10,7 @@ import (
 
 	"github.com/influxdata/influxdb/chronograf"
 	"github.com/influxdata/influxdb/chronograf/mocks"
-	"github.com/julienschmidt/httprouter"
+	"github.com/influxdata/httprouter"
```

Comment:
I thought the go.mod replace was supposed to make import changes unnecessary?

Reply:
Yeah as @mark-rushakoff says. This is great until the compat tests run and then it breaks :( but seems like bringing the fork under our banner works.

My latest problem is flux depends on the original version 😂 that is my next challenge.